controllers/numaresourcesoperator_controller.go

```diff
@@ -281,6 +282,11 @@ func (r *NUMAResourcesOperatorReconciler) syncMachineConfigPoolsStatuses(instanc
 func (r *NUMAResourcesOperatorReconciler) syncNUMAResourcesOperatorResources(ctx context.Context, instance *nropv1alpha1.NUMAResourcesOperator, mcps []*machineconfigv1.MachineConfigPool) ([]nropv1alpha1.NamespacedName, error) {
 	klog.Info("RTESync start")
 
+	errorList := r.deleteUnusedDaemonSets(ctx, instance, mcps)
+	if len(errorList) > 0 {
+		klog.ErrorS(fmt.Errorf("some errors found while deleting unused daemonsets"), "errors", errorList)
```

Comment:
`fmt.Errorf` looks redundant, and I'd drop the "some errors" part: `"failed to delete unused daemonsets", "errors", errorList`
but not worth reuploading for this

Reply:
Ok with the "some errors" part but ErrorS expect an error as first parameter so that's why I have use fmt.Errorf ... maybe I could use klog.Errorf instead and avoid the use of fmt.Errorf...

Reply:
nope, I misread, keep it this way.